sdk/keyvault/keyvault-keys/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+jobs:
+  - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-tests-node.yml
+    parameters:
+      PackageName: "@azure/keyvault-keys"
+      EnvVars:
+        AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        KEYVAULT_NAME: $(js-keyvault-keys-test-name)
+      # KeyVault Keys tests do not support concurrent execution
+      MaxParallel: 1
```

Comment:
This parameter specifies the number of independent instances of the test process to run (i.e. OS and Node version combinations)... given that these tests have things like `getUniqueName` I think multiple simultaneous runs of `integration-test:node` will complete successfully without interfering with each other. 

If that's the case for these tests, we should remove `MaxParallel` here which will run the full node matrix (https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests-node.yml#L6) in parallel. 

The only reason we'd want to keep `MaxParallel` at 1 today is if two instances of `integration-test:node` running at the same time would interfere with each other.